src/agentscope/service/web/dblp.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
+""" Search papers, authors and venues in DBLP API.
+For detail usage of the DBLP API
+please refer to https://dblp.org/faq/How+can+I+fetch+DBLP+data.html
+"""
+from agentscope.service.service_response import (
+    ServiceResponse,
+    ServiceExecStatus,
+)
+from agentscope.utils.common import requests_get
+
+
+def dblp_search_publications(
+    question: str,
+    num_results: int = 30,
+    first_hit: int = 0,
+    num_completion: int = 10,
+) -> ServiceResponse:
+    """
+    Search publications in the DBLP database
```

Comment:
If the short description is split into several lines, the ServiceFactory module may not extract the function description normally. It's suggested to test it by `ServiceFactory.get(dblp_search_publications)`